### ISSUE TITLE
glmReserve: Missing row in summary

### ISSUE THREAD

**Issue by JackyP** [mages/ChainLadder#70]
Hi,

There is an inconsistency where Mack summary has all origin periods, but glmReserve drops the first origin period.

```{r}
library(ChainLadder)

dev_glm <- glmReserve(GenIns)
dev_mack <- MackChainLadder(GenIns)

dev_glm$summary
summary(dev_mack)$ByOrigin
```

**Comment by marcopark90** [comment]
Hi,

I did not developed this function myself, however I think this is the case because the oldest AY in the triangle is assumed to be fully developed, hence IBNR is set to zero and the S.E. is set to zero as well.

**Comment by JackyP** [comment]
Yes, but wouldn't that be true for the Mack estimates as well? Ultimates for both methods are the same except for the missing first origin period.

**Comment by marcopark90** [comment]
It is true for the Mack method as well. And yes, as long you choose an odp distribution, ultimates are the same. Regarding the fact of showing or not showing the first row, it is only a style choice, it doesn't add any additional information related to IBNR or CV.

**Comment by JackyP** [comment]
I see - thanks for the quick reply! For extra context, I was looking to compare projections between different methods, potentially select one of them if results were appropriate, then take the results further for additional data processing within an R based pipeline. 

So I used the two different summary properties and functions above, which each seemed to produce the relevant summaries for the models. In doing so would I be misunderstanding the use of the methods or the goals of this package?

**Comment by marcopark90** [comment]
The goal of the package is to provide reserving methodologies and analysis in the context of loss reserving. And that's exactly what you're doing, trying different methods and compare the results, so you're spot on. Beware that the results returned will not tell you which is the "best" method. That depends on your specific case and the prior knowledge about the specific environment you're dealing with.
